### DISCUSSION TITLE
Fatal error when booting Fusion360 for first time

### DISCUSSION THREAD

**Discussion by MaNamaPaul** [cryinkfly/Autodesk-Fusion-360-for-Linux#187] [Q&A]
Hello,
I am still trying to get used to using Ubuntu and Wine, but when I follow the installation instructions(using latest nvidia graphics drivers and Wine 7.1(staging)), I keep ketting a fatal error on bootup for an exception in the materials library, specifically adexmtsv_prod.cfg.

(Exception caught in main: d:
\stage\workspace\protein\release\2020.10.r2022\protein\src\libs\xirang\common\fi
le_archive.cpp(95)MaterialLibrary/Material Library/adexmtsv_prod.cfg)

I have tried editing the file location and header/footer for Fusion but I keep getting the same error.
Any advice would be appreciated.

**Comment by cryinkfly** [comment]
Can you give me more information about your problem?
